samples/2_Cookbook/12_cmake_hip_add_executable/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 cmake_minimum_required(VERSION 2.8.3)
 
+if ($ENV{ROCM_PATH})
+    set(opt_rocm_install_path $ENV{ROCM_PATH})
+else()
+    set(opt_rocm_install_path "/opt/rocm")
+endif()
+
```

Comment:
Why is this needed? Who uses opt_rocm_install_path?

Reply:
Removed